vignettes/cli-config-user.Rmd

```diff
@@ -9,18 +9,83 @@ editor_options:
     wrap: sentence
 ---
 
-These are environment variables and options that uses may set, to modify
+These are environment variables and options that users may set, to modify
 the behavior of cli.
 
-## User facing environment variables
+## Hyperlinks
```

Comment:
It was awkward to document the options and env vars separately and caused lots of duplication. However, you can see that state if you wish: f6eec6afe50bd3759851605d2a5655e419298908

So, instead, I created a section for hyperlinks and documented all configuration at the same time.

I think there might be other big themes here (such as "colors", "progress") that could be treated the same way. What do you think? I guess what really matters is whether you're OK with hyperlinks getting this treatment.

Reply:
This is reference documentation, so I would think it is better to have an alphabetical list of env vars and another list of options, like in a dictionary. But it is also easy to search on an HTML page, so I don't really care that much.